src/Button/stories.js

```diff
@@ -20,7 +20,7 @@ storiesOf('Button')
       <figure><Button href="#" variant="primary" target="_blank">Link</Button></figure>
       <figure><Button href="#" variant="primary" disabled>Link disabled</Button></figure>
       <figure><Button href="#" variant="primary" busy='Link busy ...' disabled>Hide text by busy text</Button></figure>
-      <figure><Button href="#" variant="link" icon="success">Link style</Button></figure>
+      <figure><Button href="#" variant="link" icon="link">Link style</Button></figure>
     </div>
```

Comment:
we have change de icon to a "link" so now you could see it into a button in the storybook